catalog-scripts/idp-catalog-wizard-bitbucket.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import requests
+import os
+from requests.auth import HTTPBasicAuth
+import argparse
+from requests.packages.urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+import subprocess
+
+current_directory = os.path.basename(os.getcwd())
+branch  = "main"
+
+yaml_content_template = """
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  tags:
+    - auto-generated
+    - auto
+  name: {repo_name}
+  annotations:
+    backstage.io/source-location: url:{repo_path}
+spec:
+  type: service
+  system: {workspaceName}
```

Comment:
How about this?
domain -> workspace
system -> project
service -> repo
cc: @OrkoHunter